src/scraper/scraper.cpp

```diff
@@ -1711,12 +1714,15 @@ bool ProcessProjectRacFileByCPID(const std::string& project, const fs::path& fil
                     continue;
                 }
 
-                // Check to see if the user's team ID is in the whitelist team ID map for the project. If not continue.
+                // Check to see if the user's team ID is in the whitelist team ID map for the project.
                 for (auto const& iTeam : mTeamIDsForProject)
                 {
                     if (iTeam.second == nTeamID)
-                        continue;
+                        bOnTeamWhitelist = true;
```

Comment:
I was going to say you could also break here :)

I wish C++ had the Python equivalence of for loops. Then it would be something like:
```
for (auto const& iTeam : mTeamIDsForProject)
{
      if (iTeam.second == nTeamID)
         break;
}
else
   // Not in whitelist
   continue;
```

Reply:
I know... I could have put a break in but there is in reality only one element. My guess is that we will never do more than one team in the whitelist before we go to unfiltered.